README.md

```diff
@@ -50,7 +50,7 @@ This should make the `tyk-sync` command available to your console.
 
 ### Docker:
 
-To install particular version of `tyk-sync` via docker image please run the command bellow with appropriate version you want to use. All available versions could be found on Tyk Sync Docker Hub page here: https://hub.docker.com/r/tykio/tyk-sync/tags
+To install particular version of Tyk Sync via docker image please run the command bellow with appropriate version you want to use. All available versions could be found on Tyk Sync Docker Hub page here: https://hub.docker.com/r/tykio/tyk-sync/tags
```

Comment:
A couple of minor changes in this line, please:
`To install a particular version of Tyk Sync via docker image please run the following command stating the version you want to use. A list of all available versions can be found on the Tyk Sync Docker Hub page: https://hub.docker.com/r/tykio/tyk-sync/tags`